routes/api.php

```diff
@@ -25,3 +27,5 @@
 Route::fallback(function () {
     return redirect()->to('https://opentransportmanager.github.io/otm-docs/');
```

Comment:
To be honest, I wouldn't use docs page as 404, but maybe this is something you need to to discuss. ;)